[Notifier] [Bluesky] Allow to attach website preview card (#59667)
symfony/symfony: src/Symfony/Component/Notifier/Bridge/Bluesky/README.md

BLUESKY_DSN=bluesky://nyholm.bsky.social:p4ssw0rd@bsky.social
```

Adding Options to a Message
---------------------------

> With an `ChatMessage` Message, you can use the `BlueskyOptions` class to add message options.

Reviewer: ```suggestion
Use a `BlueskyOptions` object to add options to the message:
```

Author: Done :)

```php
use Symfony\Component\Notifier\Bridge\Bluesky\BlueskyOptions;
use Symfony\Component\Notifier\Message\ChatMessage;

$message = new ChatMessage('My message');

// Add website preview card to the message
$options = (new BlueskyOptions())
    ->attachCard('https://example.com', new File('image.jpg'))

>     // ...

Reviewer: Can you also describe `attachMedia()` as this the only other possible option.

Author: I've added the method and commented on it, as it's not possible to use both of them at the same time.
